bcbio/variation/coverage.py

```diff
@@ -226,9 +230,10 @@ def run_mosdepth(data, target_name, bed_file, per_base=False, quantize=None, thr
                 quant_export += " && "
             else:
                 quant_arg, quant_export = "", ""
-            thresholds = "-T " + ",".join([str(t) for t in thresholds]) if thresholds else ""
+
+            thresholds_cmdl = "-T " + ",".join([str(t) for t in thresholds]) if out.thresholds else ""
```

Comment:
This is causing the test failure (https://travis-ci.org/chapmanb/bcbio-nextgen/builds/294676740#L4918) but we can revert back have this as long as we have thresholds set (assume mosdepth 0.2.0 or better installed).